must-gather/collection-scripts/gather

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Usage:
+# ./gather [BASE_COLLECTION_PATH]
+#
+# where:
+#   BASE_COLLECTION_PATH is an optional argument to define the path of the collection directory, defaults to
+#   `/must-gather`
+
+
+CMA_SUB_NAME="openshift-custom-metrics-autoscaler-operator"
+
+# find namespace
+CMA_NS=$(oc get subs -A -o template --template '{{range .items}}{{if eq .spec.name "'"${CMA_SUB_NAME}"'"}}{{.metadata.namespace}}{{end}}{{end}}')
```

Comment:
@joelsmith  I'm leaning on the subscription to find the namespace where the operator was installed. If the CMA subscription is absent, this logic assumes the operator is not present; hence, there is nothing to collect. Thoughts?

Should the logic entertain other cases, the operator was deployed directly (without OLM), etc.?

Reply:
I think this is fine, especially since this is a support tool, and our supported use case is installation via OLM.